src/components/tester/TesterHome.jsx

```diff
@@ -1,61 +1,27 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import MapExplorer from '../../containers/maps/MapExplorer';
 import { connect } from 'react-redux';
-import { Actions } from 'jumpstate';
-import {
-  CLASSROOMS_STATUS, CLASSROOMS_INITIAL_STATE, CLASSROOMS_PROPTYPES,
-} from '../../ducks/classrooms';
+
+const mapConfig = require('../../lib/wildcam-darien.mapConfig.json');
```

Comment:
_Picard face palm._ Thanks for catching that - I wasn't paying attention when I did some copy pasta from WildCam Gorongosa Lab - this should be changed to `import`